scipy/stats/tests/test_multivariate.py

```diff
@@ -2908,3 +2911,172 @@ def test_random_state_property():
     for distfn, args in dists:
         check_random_state_property(distfn, args)
         check_pickling(distfn, args)
+
+
+class TestDirichletMultinomial:
+    @classmethod
+    def get_params(self, m):
+        rng = np.random.default_rng(28469824356873456)
+        alpha = rng.uniform(0, 100, size=2)
+        n = rng.integers(10, 100)
+        x = rng.integers(1, 10, size=(m, 2))
+        # ensure x sums to n, not the other way around (n must be scalar)
+        x = np.round(x / x.sum(axis=-1, keepdims=True)) * n
```

Comment:
I noticed that with this generation scheme, `x` is always 0 or 10. For example, in the `test_against_betabinom_pmf`, `x` is:

```
array([[ 0., 10.],
       [10.,  0.],
       [ 0., 10.],
       [ 0., 10.],
       [ 0., 10.],
       [ 0., 10.],
       [10.,  0.],
       [ 0., 10.],
       [10.,  0.],
       [ 0., 10.],
       [ 0., 10.],
       [10.,  0.],
       [10.,  0.],
       [ 0., 10.],
       [10.,  0.],
       [10.,  0.],
       [10.,  0.],
       [10.,  0.],
       [10.,  0.],
       [ 0., 10.],
       [10.,  0.],
       [10.,  0.],
       [ 0., 10.],
       [ 0., 10.],
       [ 0., 10.],
       [10.,  0.],
       [10.,  0.],
       [10.,  0.],
       [ 0.,  0.],
       [10.,  0.],
       ...
```

Why not use `multinomial` instead?

```suggestion
        x = rng.multinomial(n, alpha, size=(m, 2))
```

Reply:
Thanks. That was another last minute change that I didn't look at closely enough.

Reply:
Note that `x = rng.multinomial(n, alpha, size=(m,))` won't quite work because the second argument to `multinomial` must be probabilities.